Dark mode espeasy_default.css

chromoxdor opened the pull request:
- dark mode css with some optimizations for using on smartphone

chromoxdor committed:
Update espeasy_default.css

- dark mode css with some optimizations for using on smartphone

chromoxdor committed:
Update espeasy_default.css

Last minute color change for the header and menu

chromoxdor committed:
Update espeasy_default.css

corrected some minor mistakes

TD-er commented:
Ah, so my Github instructions were sufficient :)

chromoxdor commented:
Yes ,they were. Thank you!
I hope there are people enjoying this theme too. I works quite well except the rendering on iphones is somehow not right. (but nothing serious )

TD-er commented:
I will also have a look at the changes you made to see if we might also need to patch the normal theme.
And I have to look into declaring some constant values in CSS, as we only need to change a handful of colors.
So switching should be really easy them, without adding a lot more to the binary.
At least having it included as a separate esp.css file which people can upload to their node is already an option of course :)

chromoxdor committed:
Update espeasy_default.css

-menu icons are more responsive now
-minor corrections

chromoxdor commented:
Have look in the latest changes, test it and if you like what you see i can for now apply the changes to the standard css as well. (without the color changes of course)

chromoxdor commented:
and maybe we do something like this:
https://www.w3schools.com/howto/howto_js_toggle_dark_mode.asp

TD-er commented:
Or we can use CSS variables: [w3schools - CSS Variables - The var() Function](https://www.w3schools.com/css/css3_variables.asp)
The CSS can be served and at the end of the page load, I can insert a JS call to set the variables.

Grovkillen commented:
Please don't add too much stuff that my new GUI is already taking care of :)

Den tors 14 okt. 2021 08:46TD-er ***@***.***> skrev:

> Or we can use CSS variables: w3schools - CSS Variables - The var()
> Function <https://www.w3schools.com/css/css3_variables.asp>
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/letscontrolit/ESPEasy/pull/3806#issuecomment-943045152>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AGLYCYRW67HTB2Y3WMKI4H3UGZ4DRANCNFSM5F54NCRQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
>

chromoxdor committed:
Update espeasy_default.css

-some color changes
-highlight active node is back
-enhanced responsiveness for phones

chromoxdor commented:
i am curious, how can i test your new gui?

chromoxdor committed:
Update espeasy_default.css

minor mistake corrected

chromoxdor committed:
Update espeasy_default.css

-kind of disabled textarea:hover by giving it the same color. seems to be present only in rules and there it is annoying

Misiu commented:
@Grovkillen we all are waiting for updates on the new GUI :) Please don't make us wait any longer

chromoxdor committed:
Merge branch 'mega' into feature/CSSdark
